the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
 			<scope>test</scope>
 		</dependency>
-		
+
 		<!-- json-simple -->
-		
+
 		<dependency>
 			<groupId>com.googlecode.json-simple</groupId>
```

Comment:
LoginPage:
Line 22 - Use the most simple locators. Selenium has By.ID that is faster than XPath